SD_DSC.adoc

```diff
@@ -2051,11 +2053,13 @@ If applicable, the evaluator shall confirm that the guidance documentation conta
 
 The evaluator shall perform the following tests:
 
-Test 1: The evaluator shall ensure that all communications between distributed TOE components is tested during the course of the evaluation, setting up the connections as described in the guidance documentation (if applicable) and ensuring that communication is successful.
+Test 1: The evaluator shall ensure that all communications between distributed TOE components are tested. If communications channels require configuration, the guidance documentation shall be followed and ensure that communication is successful.
 
 Test 2: The evaluator shall ensure, for each communication channel, that the channel data is not sent in plaintext or that there is no interface by which it is possible to extract the channel data.
 
-Test 3: The evaluator shall, where possible, physically interrupt communications between TOE components. The evaluator shall ensure that when physical connectivity is restored, communications are appropriately protected. Note that this test shall only be performed for those interfaces where physical interruption of communications is typical operational behavior. If the TOE consists of multiple components embedded within the same physical chassis such that interrupting communications requires physical destruction of the device (as opposed to disconnecting a re-connectable wire or temporarily disabling a radio), this test is not applicable.
+Test 3 [conditional]: The evaluator shall, where possible, physically interrupt communications between TOE components. The evaluator shall ensure that when physical connectivity is restored, communications are appropriately protected. 
```

Comment:
Is this test supposed to address the optional protection against modification that can be selected in FPT_ITT.1? If so, I don't think this test is sufficient.

There should be a test where the evaluator ensures that the TOE detects any accidental or intentional modifications to the data, if possible (i.e. if the evaluator can act as a MiTM). I believe this is different from physically interrupting the communication channel.

Reply:
@woodbe I don't think there is such a test for this specific scenario (internal data transfer). I agree with you on the test you modified, so I think that one should stay.

I added a commit that adds a new test as test 3 (for aesthetic reasons). The old test 3 is now test 4.

Reply:
Had to force push to fix my email address...

Reply:
Ok, I think this is good